src/components/pastIncidents/PastIncidents.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { Meta, StoryFn } from "@storybook/react";
+import { PastIncidents, PastIncidentsData } from "./PastIncidents";
+import jsonData from "../../../public/incidents.json";
+
+const typedJsonData: PastIncidentsData["data"] =
+  jsonData as PastIncidentsData["data"];
+
+export default {
+  title: "Component/PastIncidents",
+  component: PastIncidents,
+} as Meta<typeof PastIncidents>;
+
+const Template: StoryFn<typeof PastIncidents> = (args) => (
+  <PastIncidents {...args} />
+);
+
+export const Simple = Template.bind({});
+Simple.args = {
+  title: "Past Incidents",
+  subTitle: "Showing all past incidents in the last 6 months",
+  data: typedJsonData,
```

Comment:
Lets put the data in the story as an arg, i.e. omit import jsonData from "../../../public/incidents.json"; This will keep the story static

Reply:
Updated